README.md

```diff
@@ -30,131 +30,69 @@
 
 </div>
 
-# Beta9
+### Beta9 is an open source platform for running multi-cloud GPU workloads
 
-Beta9 makes it easy for developers to run serverless functions on cloud GPUs.
+- Run workloads on any VM, anywhere
+- Read files at the edge from your local filesystem using distributed, cross-region storage
+- Connect your fleet of GPUs using a Tailscale-powered service mesh
+- Provision new VMs from any cloud using a single cURL command
```

Comment:
This sounds a bit confusing. We don't provision VMs. We run workloads on already existing VMs that are connected to the tailnet